include/libfreenect2/packet_pipeline.h

```diff
@@ -66,6 +66,13 @@ class LIBFREENECT2_API PacketPipeline
   PacketPipelineComponents *comp_;
 };
 
+ class LIBFREENECT2_API DumpPacketPipeline: public PacketPipeline
+ {
+ public:
+   DumpPacketPipeline();
+   virtual ~DumpPacketPipeline();
+ };
```

Comment:
So in the public API you didn't actually expose the interfaces to export the P0, X, Z, LUT tables.

Just add `get*Tables()`? What is your plan?